src/main/java/com/thebois/models/inventory/items/Item.java

```diff
@@ -21,4 +24,17 @@ public ItemType getType() {
         return type;
     }
 
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) return true;
+        if (o == null) return false;
+        final Item item = (Item) o;
+        return type == item.type;
```

Comment:
Style
I'm pretty sure we said the only variable names that were allowed to be 1 char long is x and y.

Reply:
We have? Those I've written have used `object`, mostly because Jonathan pointed them out.

Reply:
I changed all parameters named o to other